modules/sno-du-enabling-kdump.adoc

```diff
@@ -0,0 +1,30 @@
+// Module included in the following assemblies:
+//
+// * scalability_and_performance/ztp-far-edge/ztp-reference-cluster-configuration-for-vdu.adoc
+
+:_content-type: CONCEPT
+[id="sno-du-enabling-kdump_{context}"]
+= kdump
+
+`kdump` is a Linux kernel feature that creates a kernel crash dump when the kernel crashes. You can enable `kdump` with the following `MachineConfig` custom resource (CR):
+
+.kdump recommended configuration
+[source,yaml]
+----
+apiVersion: machineconfiguration.openshift.io/v1
+kind: MachineConfig
+metadata:
+  labels:
+    machineconfiguration.openshift.io/role: master
+  name: 06-kdump-enable-master
+spec:
+  config:
+    ignition:
+      version: 3.2.0
+    systemd:
+      units:
+      - enabled: true
+        name: kdump.service
+  kernelArguments:
+    - crashkernel=256M
```

Comment:
This should be 512M.
https://github.com/openshift-kni/cnf-features-deploy/blob/release-4.11/ztp/source-crs/extra-manifest/06-kdump-master.yaml#L18

Reply:
Actually this file is deleted. Update is made in https://github.com/openshift/openshift-docs/pull/49805/commits/1a96469bb0aa9b413ffe0d1f0d404b8895b1cc23#diff-7918e91fa3206dfdde579390fcff3e13d04a5cf57b2d007fcf037d1a86dab07d